<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config Maker v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="subTestListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>260</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="usedSubTestListView">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>260</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="useSubTestButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use the selected subTest&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeSubTestButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove subTest&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fileListComboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel </string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New </string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save </string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As ..</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
